tools/internal_ci/linux/grpc_distribtests_python.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+# Copyright 2021 The gRPC Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -ex
+
+# change to grpc repo root
+cd $(dirname $0)/../../..
+
+source tools/internal_ci/helper_scripts/prepare_build_linux_rc
+
+# some distribtests use a pre-registered binfmt_misc hook
+# to automatically execute foreign binaries (such as aarch64)
+# under qemu emulator.
+source tools/internal_ci/helper_scripts/prepare_qemu_rc
+
+# Build all python linux artifacts (this step actually builds all the binary wheels and source archives)
+tools/run_tests/task_runner.py -f artifact linux python -j 6 -x build_artifacts/sponge_log.xml || FAILED="true"
```

Comment:
Will the other steps below fail fast if the artifacts fail to build? It might be worth exiting here on failure to speed things up.

Reply:
I think we quite often have situations where only one of the artifact builds gets broken, but the others will build just fine. Running the distribtests in that case gives you a good picture which distribtests are impacted by that and which are not (and I think that's a useful signal). 
AFAIK the distribtests will fail quite fast (in like a minute is so) when the wheel required by a particular distribtest is missing.